src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Shape {
+    private int height;
+    private int sideA;
+    private int sideB;
+
+    public IsoscelesTrapezoid(int height, int sideA, int sideB, String color) {
+        super(color);
+        this.height = height;
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    public void setParameters(int height, int sideA, int sideB) {
```

Comment:
let's remove this method - if you wish you can create getters and setters for fields

Reply:
done